addons/web/static/src/js/fields/field_wrapper.js

```diff
@@ -33,19 +33,24 @@ odoo.define('web.FieldWrapper', function (require) {
             this._data.res_id = record.res_id;
             this._data.model = record.model;
             this._data.mode = options.mode || "readonly";
-            this._data._isValid = true;
-            this._data.lastSetValue = undefined;
             this._data.formatType = this._data.attrs.widget in field_utils.format ?
                                 this._data.attrs.widget :
                                 this._data.field.type;
             this._data.formatOptions = {};
             this._data.parseOptions = {};
-            if (this._data.attrs.decorations) {
-                this._data.resetOnAnyFieldChange = true;
-            }
-            for (const key in this._data) {
+
+            for (let key in this._data) {
                 Object.defineProperty(this, key, {
-                    get: () => (this.el ? this.componentRef.comp : this._data)[key],
+                    get: () => {
+                        if (this.el) {
+                            if (key === 'dataPointID') {
+                                return this.componentRef.comp.dataPointId;
+                            } else if (key === 'res_id') {
+                                return this.componentRef.comp.resId;
+                            }
```

Comment:
not sure we want to do this, as we already discussed (if I remember correctly)

Reply:
wouldn't do that (here at least). if you really want to do that, the right place is the wrapper i think

Reply:
ho my bad :D